modules/database/models/recently_added_item.py

```
@@ -0,0 +1,20 @@
+from modules.database.base.imports import *
+
+from modules.utils import get_now_timestamp
+
+
+class RecentlyAddedItem(Base):
+    __tablename__ = 'recently_added_items'
+    id = Column("id", Integer, primary_key=True, autoincrement=True, nullable=False)
+    name = Column("name", String, nullable=False)
```

Ruff (0.8.0)

`Column` may be undefined, or defined from star imports

(F405)